Moco/exampleWalking/exampleWalking.m

```diff
@@ -0,0 +1,284 @@
+% OpenSim Moco: exampleWalking.m                                             %
+% -------------------------------------------------------------------------- %
+% Copyright (c) 2022 Stanford University and the Authors                     %
+%                                                                            %
+% Author(s): Carmichael Ong, Antoine Falisse, Scott Uhlrich                  %
+%                                                                            %
+% Licensed under the Apache License, Version 2.0 (the "License"); you may    %
+% not use this file except in compliance with the License. You may obtain a  %
+% copy of the License at http://www.apache.org/licenses/LICENSE-2.0          %
+%                                                                            %
+% Unless required by applicable law or agreed to in writing, software        %
+% distributed under the License is distributed on an "AS IS" BASIS,          %
+% WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.   %
+% See the License for the specific language governing permissions and        %
+% limitations under the License.                                             %
+% -------------------------------------------------------------------------- %
+
+% Example using OpenSim Moco to track OpenCap data of walking. This example
+% solves in about 5 hours on a computer with 8 cores.
+%
+% Model
+% -----
+% This script uses the same base model from OpenCap and then adjusts the
+% model to use a muscle class better suited for use with Moco
+% (DeGrooteFregly2016), to increase the optimal force of
+% CoordinateActuators that actuate the lumbar and upper limb joints, and to
+% weld the toe (mtp) joints.
+%
+% Data
+% ----
+% Data are from a processed OpenCap trial of walking and contain coordinate
+% values after inverse kinematics.
+
+clear;
+
+% Load the Moco libraries
+import org.opensim.modeling.*;
+
+% ---------------------------------------------------------------------------
+% Set up a coordinate tracking problem where the goal is to minimize the
+% difference between provided and simulated coordinate values and speeds, 
+% as well as to minimize an effort cost (squared controls). Endpoint
+% constraints enforce periodicity of the coordinate values (except for
+% pelvis tx) and speeds, coordinate actuator controls, and muscle 
+% activations.
+
+
+% Define the optimal control problem
+% ==================================
+mocoTrackName = 'walking1';
+track = MocoTrack();
+track.setName(mocoTrackName);
+
+% Set the weights for the terms in the objective function. The values below were
+% obtained by trial and error.
+controlEffortWeight = 0.1;
+stateTrackingWeight = 1;
+
+% Set other parameters, including the initial and final times of the
+% simulation, the mesh interval, the relative size of the state bounds, and
+% the optimal force for the lumbar and upper limb coordinate actuators.
+initialTime = 0.07;
```

Comment:
maybe a little messy, but could be good to comment here and in the squats one how we select these times for periodicity constraints